torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -588,6 +598,31 @@ class TensorExprKernel {
             });
       } break;
 
+      case aten::sigmoid: {
```

Comment:
@bertmaher , @resistor Can you guys please take a quick look at Relu, Neg, Reciprocal and Sigmoid, and let me know if it looks okay

Reply:
@bertmaher @resistor  can you guys take another look at this reimplementation. For Sigmoid small fp differences show up, for which I had to change rtol and atol values in numpy.allclose()